src/pages/resultsView/expression/ExpressionWrapper.tsx

```diff
@@ -541,21 +543,25 @@ export default class ExpressionWrapper extends React.Component<ExpressionWrapper
             >
                 <div className="arrow" style={{top: 27}}></div>
                 <div className="popover-content">
+                    <strong>Study:</strong> {this.tooltipModel!.data.studyName}<br/>
```

Comment:
we could build a better method of listing this stuff out but bigger fish to fry right now